2021/31-brendan-bokeh.md

```diff
@@ -15,9 +15,9 @@
 - Bokeh notebooks:  https://github.com/bokeh/bokeh-notebooks
 - Bokeh docs:  https://docs.bokeh.org/en/latest/index.html
 - Starting from timestamp 16:48, Brendan is working on the Bokeh Quickstart which is at https://github.com/bokeh/bokeh-notebooks/blob/main/quickstart/quickstart.ipynb.
-- Then he starts on the tutorial from 30:34 https://github.com/bokeh/bokeh-notebooks/tree/main/tutorial 
+- Then he starts on the tutorial from 30:34 https://github.com/bokeh/bokeh-notebooks/tree/main/tutorial
 
-## Video:   ** NEED TO UPDATE **
+## Video:
 
 <a href="http://www.youtube.com/watch?feature=player_embedded&v=gMchDJP0yEI" target="_blank"><img src="http://img.youtube.com/vi/XKNdXN-Jfmo/0.jpg" 
```

Comment:
@nestornav can you double check the video link in this code? I don't think it's the right one.

Reply:
The video is right, I didn't check the image about it. Sorry